src/dagre/index.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import DagreGraph from 'dagre-d3-react'
+import useResizeAware from 'react-resize-aware'
+
+
+export const stepProps = Object.freeze({
+  audience_build_wi: {
+    name: 'Audience Build: Walk-in',
+    level: 0,
+  },
+  audience_build_beacon: {
+    name: 'Audience Build: Beacon',
+    level: 0,
+  },
+  audience_enrich_aoi: {
+    name: 'Enrichment: AOI',
+    level: 1,
+  },
+  audience_enrich_xd: {
+    name: 'Enrichment: Cross Device',
+    level: 1,
+  },
+  audience_intersect_vwi: {
+    name: 'Intersection: Verified Walk-in',
+    level: 2,
+  },
+  audience_intersect_xwi: {
+    name: 'Intersection: Cross Chain Walk-in',
+    level: 2,
+  },
+  segment: {
+    name: 'Segment',
+    level: 3,
+  },
+  cohort_repeat_visits: {
+    name: 'Cohort Analysis: Visits',
+    level: 3,
+  },
+  cohort_converted_visitors: {
+    name: 'Cohort Analysis: Converted Visitors',
+    level: 3,
+  },
+  propensity: {
+    name: 'Propensity',
+    level: 3,
+  },
+  report_wi: {
+    name: 'Report: Walk-in',
+    level: 3,
+  },
+  report_vwi: {
+    name: 'Report: Verified Walk-in',
+    level: 3,
+  },
+  report_xwi: {
+    name: 'Report: Cross Chain Walk-in',
+    level: 3,
+  },
+})
+
+const humanTime = (seconds) => {
+  let cd = seconds
+  const time = {}
+  time.day = Math.floor(cd / 86400)
+  cd -= time.day * 86400
+  time.hour = Math.floor(cd / 3600)
+  cd -= time.hour * 3600
+  time.minute = Math.ceil(cd / 60)
+  const parts = []
+  if (time.day) {
```

Comment:
could create an anonymous function so if this ever needs to be changed, its easier to do so

```
const convertToReadable = (view) => {
  return parts.push(`${time[view]} ${view}${time[view] > 1 ? 's' : ''}`)
}
```